dotnet/src/SemanticKernel/Planning/SKContextPlanningExtensions.cs

```diff
@@ -79,7 +79,7 @@ internal static class SKContextPlanningExtensions
             await RememberFunctionsAsync(context, availableFunctions);
 
             // Search for functions that match the semantic query.
-            var memories = context.Memory.SearchAsync(PlannerMemoryCollectionName, semanticQuery!, config.MaxRelevantFunctions, config.RelevancyThreshold.Value,
+            var memories = context.Memory.SearchAsync(PlannerMemoryCollectionName, semanticQuery!, config.MaxRelevantFunctions, config.RelevancyThreshold.Value, false,
```

Comment:
Could we add parameter labels to this call for clarity?

Reply:
done